src/App/Pages/LoginPage.cs

```diff
@@ -67,7 +67,7 @@ private void Init()
 
             var table = new ExtendedTableView
             {
-                Intent = TableIntent.Settings,
+                Intent = TableIntent.Form,
```

Comment:
Not sure how this will affect others.

Reply:
This isn't required, it just seemed to make more sense then settings.

Reply:
This should be reverted.

Reply:
Ok, I can remove the patch